src/core/lib/security/transport/security_handshaker.c

```diff
@@ -146,6 +147,18 @@ static void on_peer_checked(grpc_exec_ctx *exec_ctx, void *arg,
     security_handshake_failed_locked(exec_ctx, h, error);
     goto done;
   }
+  // Create zero-copy grpc protector, if implemented.
+  tsi_zero_copy_grpc_protector *grpc_protector = NULL;
```

Comment:
Since the old-style frame protector will never be used if the zero-copy protector exists, we might as well avoid creating the old-style protector in that case.

Reply:
DONE.